app/views/includes/scripts_ga_2.html

```diff
@@ -1,7 +1,6 @@
 {% if globals.enableTracking == "true" %}
-    <!-- Google Tag Manager (noscript) -->
-    <noscript>
-        <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KG8XT3S" height="0" width="0" style="display:none;visibility:hidden" title="gtm"></iframe>
-    </noscript>
-    <!-- End Google Tag Manager (noscript) -->
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KG8XT3S"
+                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<!-- End Google Tag Manager (noscript) -->
 {% endif %}
```

Comment:
Undo all of this.

Reply:
done

Reply:
Did you push? I still see these changes.